tests/main/confinement-classic/task.yaml

```diff
@@ -1,10 +1,13 @@
 summary: trivial snap with classic confinement runs correctly
+
+systems: [-ubuntu-core-16-*, -fedora-*]
```

Comment:
Not sure why the diff is like this, I just see one list 
systems: [-ubuntu-core-*, -fedora-*]
I'll push again, I think it was a problem with the merge.